core/src/test/scripts/unit_tests_validate_data.py

```diff
@@ -76,14 +93,143 @@ def test_column_order_validation_ClinicalValidator(self):
 
         # set level according to this test case:
         logger = getNewLogger()
-        logger.setLevel(logging.WARNING)
-        validator = ClinicalValidator(hugo_entrez_map,logger,meta_dict)
-        validator.validate()
-        # we expect no errors or warnings
-        self.assertEqual(0, len(logger.handlers[0].buffer))
-        # if the file has another order, this is also OK:
-        meta_dict['data_file_path'] = 'data_clin_order2.txt'
-        validator = ClinicalValidator(hugo_entrez_map,logger,meta_dict)
+        try:
```

Comment:
please align this with the new way you used in your tests and remove getNewLogger instead of keeping 2 implementations that do the same thing